Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Contracts.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Scripts.EVM.Token
+{
+    public class Contracts
+    {
+        public const string Erc20 = "0x358969310231363CBEcFEFe47323139569D8a88b";
```

Comment:
are these token contracts? I'm asking if these values can ever change?

Reply:
Yes they are, but they're what all the samples & tests reference. Instead of declaring them over and over I just made them constant and placed them there. They should only change when we change default chains for the SDK or if we want to add more test functionality.

Reply:
And these contracts are present on Sepolia right?

Reply:
yes these are sepolia deployments